### DISCUSSION TITLE
Add $_root_url Global variable

### DISCUSSION THREAD

**Discussion by ballab1** [grafana/grafana#43233] [Dashboards, Panels & Visualizations]
Having a global variable for the 'grafana.ini' key **root_url** would allow dashboards to contain links to other dashboards in the same instance.

Currently, our setup uses links to improve navigation, however since we do not allow anyone to make changes on our production server, they must make the changes on a private Grafana instance, and then submit a Pull Request to get the changes into our production Grafana instance. When a dashboard contains a link to another dashboard, it currently gets redirected to production. It also makes it difficult when creating dashboards that reference other dashboards

**Comment by jeanlst** [comment]
Any news on this?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
